code/client/src/core/states/game_setup.cpp

```diff
@@ -0,0 +1,62 @@
+#include "game_setup.h"
+
+#include "states.h"
+
+#include "sdk/ue/game/traffic/c_streaming_traffic_module.h"
+#include "sdk/ue/gfx/environmenteffects/c_gfx_environment_effects.h"
+
+#include <utils/states/machine.h>
+
+namespace MafiaMP::Core::States {
+    GameSetupState::GameSetupState() {}
+
+    GameSetupState::~GameSetupState() {}
+
+    int32_t GameSetupState::GetId() const {
+        return StateIds::GameReady;
+    }
```

Comment:
Maybe the state should be `GameSetup` then?

To be changed in:
- code/client/src/core/states/states.h
- code/client/src/core/states/initialize.cpp